samples/highcharts/annotations-advanced/fibonacci-timezones/demo.js

```diff
@@ -0,0 +1,60 @@
+Highcharts.getJSON('https://demo-live-data.highcharts.com/aapl-c.json', function (data) {
+    var length = data.length;
+
+    // Create the chart
+    Highcharts.stockChart('container', {
+
+        chart: {
+            events: {
+                load: function () {
+                    this.annotations.forEach(function (annotation) {
+                        annotation.setControlPointsVisibility(true);
+                        annotation.cpVisibility = true;
```

Comment:
Do we need this `cpVisibility` property? Perhaps it would be better to make this a variable, so developers won't get the wrong idea that `cpVisibility` can change anything

Reply:
Sorry, could you elaborate or explain in other words?

Reply:
Sure! Is `Annotation.cpVisibility` necessary? Or can it be a simple variable: `let cpVisibility`?